netbeans/CheckerFrameworkPlugin/release/checkerstrings.properties

```diff
@@ -0,0 +1,18 @@
+org.checkerframework.checker.nullness.NullnessChecker=Nullness Checker
```

Comment:
I ordered the elements in the same order as they appeared in the Eclipse plugin, but I'm not sure if the ordering there has some signficance or not.